delete-indices.py

```diff
@@ -23,13 +23,19 @@
 hostname = os.environ['ES_HOSTNAME']
 sre_managed = os.environ['SRE_MANAGED']
 index_deleted = "index_not_found_exception"
+env = os.environ['ENV']
 
+if env == "stage": 
```

Comment:
This will certainly work, but it might be easier in the long term to make `retention_days` a parameter. That way you could set the default to `14` and then override the parameter in the stage deployment.

Reply:
that's actually a smarter way - lemme implement it